### PR TITLE
Add a profile for the staff page

### LINKED ISSUE #28: Create a proile page for staff
Currently staff cannot log out manually, which is annoying :)

### PR DESCRIPTION
**Pull request by feeds**
Closes #28
Depends on #20

### CODE REVIEW

File: backend/apps/matching/templates/staff/staff_profile.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            <path d="M2 3h10v2H2V3zm0 3h4v3H2V6zm0 4h4v1H2v-1zm0 2h4v1H2v-1zm5-6h2v1H7V6zm3 0h2v1h-2V6zM7 8h2v1H7V8zm3 0h2v1h-2V8zm-3 2h2v1H7v-1zm3 0h2v1h-2v-1zm-3 2h2v1H7v-1zm3 0h2v1h-2v-1z"/>
+        </svg>
+        {% blocktrans %}Do something{% endblocktrans %}</a>
+    </p>
+    <p>
+        <a role="button" class="btn blue text-white btn-md" href="/use_statistics/view">
+        <svg class="bi bi-bar-chart" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" d="M4 11H2v3h2v-3zm5-4H7v7h2V7zm5-5h-2v12h2V2zm-2-1a1 1 0 00-1 1v12a1 1 0 001 1h2a1 1 0 001-1V2a1 1 0 00-1-1h-2zM6 7a1 1 0 011-1h2a1 1 0 011 1v7a1 1 0 01-1 1H7a1 1 0 01-1-1V7zm-5 4a1 1 0 011-1h2a1 1 0 011 1v3a1 1 0 01-1 1H2a1 1 0 01-1-1v-3z" clip-rule="evenodd"/>
+        </svg>
+        {% blocktrans %}Do something{% endblocktrans %}</a>
```
Comment (bjrne) [inline backend/apps/matching/templates/staff/staff_profile.html:62]: Shouldn't this already be adjusted?
```suggestion
        {% blocktrans %}Show usage stats{% endblocktrans %}</a>
```

**Review by bjrne — APPROVED** [review]
Not sure about this one label, but I guess you we're either lazy or clever and had something in mind ;)

File: backend/apps/matching/templates/staff/staff_profile.html (hunk 1 above)
Comment (feeds) [reply backend/apps/matching/templates/staff/staff_profile.html:62]: At some point somebody will have to make the page pretty anyway and set the right links :)

Comment (Baschdl) [reply backend/apps/matching/templates/staff/staff_profile.html:62]: Using placeholders where there is no function is probably the clearer version but having the possibility to view the statistics would be good

**Pull request merged**